test/api.js

```diff
@@ -137,6 +137,41 @@ describe('api', function() {
       });
     });
 
+    it('should check SASS_PATH in the specified order', function(done) {
+      var src = read(fixture('include-path/index.scss'), 'utf8');
+      var expected = read(fixture('include-path/expected.css'), 'utf8').trim();
+
+      var separator = process.platform == 'win32' ? ';' : ':';
+      process.env.SASS_PATH = fixture('include-path/functions');
```

Comment:
I started writing the code to save any initial value for SASS_PATH, then set it back to what it was afterwards. But then I thought it's actually quite unlikely that any changes to `process.env` would effect the environment outside of this particular `mocha` cycle, so I went to test it:

``` bash
$ export SASS_PATH=hello
$ mocha test/api.js
...
 129 passing (255ms)
$ echo $SASS_PATH
hello
```

So I think all I need to do it make sure SASS_PATH changes don't bleed through from one test to the other. So I've reset SASS_PATH to be an empty string before each test.